lib/ionic_notification/sent_notification.rb

```diff
@@ -0,0 +1,30 @@
+module IonicNotification
+  class SentNotification
+    attr_accessor :tokens, :title, :message, :android_payload,
+      :ios_payload, :production, :result, :message_id, :sent_at
+
+    def initialize(options = {})
+      @tokens = options[:tokens]
+      @title = options[:title]
+      @message = options[:message]
+      @android_payload = options[:android_payload]
+      @ios_payload = options[:ios_payload]
+      @production = options[:production]
+      @result = options[:result]
+      @message_id = options[:message_id]
+      @sent_at = Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.